.github/workflows/soak.yml

```diff
@@ -316,7 +305,7 @@ jobs:
                                      --erratic-soaks ${{ needs.compute-soak-meta.outputs.erratic-soaks }}
 
   soak-comparison:
-    name: Soak (${{ matrix.target }}) - comparison - replica ${{ matrix.replica }}
+    name: Soak (${{ matrix.target }}) - comparison
```

Comment:
This can be a follow-up, but would it make sense to run the comparison and baseline on the same instance to reduce noise?

Reply:
I did wonder about that. It's something I'll experiment with when I start working on tracking down the noise issue we're still having.